Harvest.Web/Startup.cs

```diff
@@ -136,7 +136,9 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<FinancialLookupSettings>(Configuration.GetSection("FinancialLookup"));
             services.Configure<SlothSettings>(Configuration.GetSection("Sloth"));
             services.Configure<SparkpostSettings>(Configuration.GetSection("SparkPost"));
+            services.Configure<StorageSettings>(Configuration.GetSection("Storage"));
 
+            services.AddSingleton<IFileService, FileService>();

```

Comment:
Why does this need to be a singleton?

Reply:
Doesn't need to be but it has no state so I figured why bother recreating it on every request.  Though perhaps that isn't a good enough reason and since the others are all scoped I could do the same